### DISCUSSION TITLE
More control around backing up RAW/HEIF/JPEGs

### DISCUSSION THREAD

**Discussion by 2001Y** [immich-app/immich#12822] [Feature Request]
### The bug

Thank you for the great service.
I am Japanese and I am surprised to know that the logo of this service is related to Japan.

Now, currently, if you shoot Raw on iOS, immich will back up DNGs.
However, I think edited HEIF/JPEGs should be backed up.

Because the Photos app on iOS will show the edited HEIF/JPEGs.
Also, after editing images on the train on the way home,
I think it is the 'edited data' that should be backed up when you go home and connect to Wi-Fi.

If you want, both the edited HEIF/JPEG + RAW data should be saved.
This requires the concept of "stacks", which is already present in the Feature Request.

#2479 [Feature] Stacked Photos

### The OS that Immich Server is running on

Ubuntu 22.04.4 LTS

### Version of Immich Server

v1.103.1

### Version of Immich Mobile App

v1.103.0 build. 155

### Platform with the issue

- [X] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
N/A
```

**Comment by bo0tzz** [comment]
This might be a duplicate of  #9250?

**Comment by 2001Y** [comment]
@bo0tzz 
If #9250 can be solved, the same mechanism could be used to solve this issue, but the purpose of this issue is "Shouldn't HEIF/JPEG be preferred over RAW on the iPhone, even if it can't be stacked or updated after editing?" This is the story.

**Comment by jrasm91** [comment]
Backup to immich is based on folders at the moment. You can put RAWs and non-RAWs in separate files if you want to have more control over which one is backed up and which on is not.
